modules/ossm-installing-multi-primary-multi-network-topology.adoc

```diff
@@ -0,0 +1,292 @@
++
+[source,terminal]
+----
+$ oc wait --context "${CTX_CLUSTER2}" --for=condition=Ready istio/default --timeout=3m
+----
+
+.. Create an East-West gateway on the West cluster by running the following command:
++
+[source,terminal]
+----
+$ oc apply --context "${CTX_CLUSTER2}" -f https://raw.githubusercontent.com/istio-ecosystem/sail-operator/main/docs/multicluster/east-west-gateway-net2.yaml
+----
+
+.. Expose the services through the gateway by running the following command:
++
+[source,terminal]
+----
+$ oc --context "${CTX_CLUSTER2}" apply -n istio-system -f https://raw.githubusercontent.com/istio-ecosystem/sail-operator/main/docs/multicluster/expose-services.yaml
+----
+
+. Install a remote secret on the East cluster that provides access to the API server on the West cluster by running the following command:
++
+[source,terminal]
+----
+$ istioctl create-remote-secret --context="${CTX_CLUSTER2}" --name=cluster2 | \
+     oc apply -f - --context="${CTX_CLUSTER1}" 
+----
+
+. Install a remote secret on the West cluster that provides access to the API server on the East cluster by running the following command:
++
+[source,terminal]
+----
+$ istioctl create-remote-secret --context="${CTX_CLUSTER1}" --name=cluster1 | \
+     oc apply -f - --context="${CTX_CLUSTER2}"
+----
+
+. Deploy sample applications on the East cluster:
+
+.. Create a sample application namespace on the East cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get ns sample --context "${CTX_CLUSTER1}" || oc create --context="${CTX_CLUSTER1}" namespace sample
+----
+
+.. Label the application namespace to support sidecar injection by running the following command:
++
+[source,terminal]
+----
+$ oc label --context="${CTX_CLUSTER1}" namespace sample istio-injection=enabled
+----
+
+.. Deploy the `helloworld` application:
+
+... Create the `helloworld` service by running the following command:
++
+[source,terminal]
+----
+oc apply --context="${CTX_CLUSTER1}" \
+  -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/helloworld/helloworld.yaml \
+  -l service=helloworld -n sample
+----
+
+... Create the `helloworld-v1` deployment by running the following command:
++
+[source,terminal]
+----
+oc apply --context="${CTX_CLUSTER1}" \
+  -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/helloworld/helloworld.yaml \
+  -l version=v1 -n sample
+----
+
+.. Deploy the `sleep` application by running the following command:
++
+[source,terminal]
+----
+$ oc apply --context="${CTX_CLUSTER1}" -n sample \
+    -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/sleep/sleep.yaml
+----
+
+. Wait for the sample applications on the East cluster to return the "Ready" status condition by running the following command:
++
+[source,terminal]
+----
+$ oc --context="${CTX_CLUSTER1}" wait --for condition=available -n sample \
+     deploy/helloworld-v1 deploy/sleep
+----
```

Comment:
IMO it would be better to move this step one level lower so that it becomes step 6.e.

Reply:
done